backends/graphics3d/openglsdl/openglsdl-graphics3d.cpp

```diff
@@ -592,9 +609,9 @@ void OpenGLSdlGraphics3dManager::updateScreen() {
 	if (_frameBuffer) {
 		_frameBuffer->detach();
 		_surfaceRenderer->prepareState();
-		glViewport(0, 0, _overlayScreen->getWidth(), _overlayScreen->getHeight());
+		glViewport(_gameDrawRect.left, _windowHeight - _gameDrawRect.top - _gameDrawRect.height(), _gameDrawRect.width(), _gameDrawRect.height());
```

Comment:
And here `_gameDrawRect.bottom`

Reply:
Same here. This looks to be less readable than the original code, though, and the optimization done won't make such a difference, IMHO